src/components/Dashboard/Form/Button/Button.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import styles from "./Button.scss";
+import PropTypes from "prop-types";
+
+const Button = props => (
+  <div>
+    <button type={props.type} className={styles.Button} onClick={props.onClick}>
+      {props.label}
+    </button>
+  </div>
```

Comment:
if you're returning just one element, you don't need `<div> </div>`  in case of returning multiply elements it's better to use React.Fragment instead div, if it's just "dumb" container without any styling etc.